README.md

```diff
@@ -2,9 +2,9 @@
 
 go-github is a Go client library for accessing the [GitHub API][].
 
-**Documentation:** [![GoDoc](https://godoc.org/github.com/google/go-github?status.svg)](https://godoc.org/github.com/google/go-github)  
-**Mailing List:** [go-github@googlegroups.com](https://groups.google.com/group/go-github)  
-**Build Status:** [![Build Status](https://travis-ci.org/google/go-github.svg?branch=master)](https://travis-ci.org/google/go-github)  
+**Documentation:** [![GoDoc](https://godoc.org/github.com/google/go-github?status.svg)](https://godoc.org/github.com/google/go-github)
```

Comment:
please revert these. the trailing whitespace is meaningful... it's converted to a br tag.